components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/policy/publisher/PolicyPublisher.java

```diff
@@ -145,177 +132,6 @@ public void publishPolicy(String[] policyIds, String version, String action, boo
         executor.run();
     }
 
-
-    public void persistSubscriber(PublisherDataHolder holder, boolean update) throws EntitlementException {
```

Comment:
Whats the reason to remove these methods and move to RegistrySubscriberDAOImpl class

Reply:
We need to separate these methods into a specific implementation of an interface that handles subscriber operations, to facilitate the database implementation.